src/Services/Chat/Chat.Infrastructure/ConfigureServices.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Chat.Application.Abstractions;
+using Chat.Application.Abstractions.Contexts;
+using Chat.Domain.Repositories;
+using Chat.Infrastructure.Contexts;
+using Chat.Infrastructure.Options.MongoDb;
+using Chat.Infrastructure.Repositories;
+using Chat.Infrastructure.Services;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace Chat.Infrastructure;
+
+public static class ConfigureServices
+{
+    public static IServiceCollection AddInfrastructureServices(this IServiceCollection services)
+    {
+        services.ConfigureOptions<MongoDbOptionsSetup>();
+
+        services.AddSingleton<IApplicationDbContext, ApplicationDbContext>();
+        services.AddScoped<IUserContext, UserContext>();
+
+        services.AddScoped<IMessageNotificationService, MessageNotificationService>();
+
+        services.AddScoped<IUserRepository, UserRepository>();
+        services.AddScoped<IConversationRepository, ConversationRepository>();  
+        services.AddScoped<IMessageRepository, MessageRepository>();  
+        services.AddScoped<IItemRepository, ItemRepository>();
```

Comment:
It can be simplified. Example: 
services.AddScoped<IUserContext, UserContext>()
.AddScoped<IMessageNotificationService, MessageNotificationService>()
...